src/content/en/2021/css.md

```diff
@@ -754,18 +864,29 @@ featured_stat_label_3: TODO
   sql_file="gradient_functions.sql"
 ) }}
 
+When prefixed versions of gradients (e.g., `-webkit-linear-gradient`) were included, the resulting graph looked basically the same as last year’s.
+
+Some other things we found in analyzing gradient values:
 
+* The median number of color stops in gradients is just two, except at the 90th percentile, where the four stops was the median.
+* Hard color stops — that is, gradients where two color stops were placed at the same position — occurred in just over half of all gradients.
+* Color-stop interpolation (a.k.a “midpoints”) were used in 21% of all gradient instances.
+
+We also saw a dramatic reduction at the top end of gradient complexity.  Last year, the gradient with the largest number of color stops had 646 stops.  This year, the winner had only 81 color stops.
 
 {{ figure_markup(
   image="most-color-stops.png",
   classes="height-16vw-122px",
-  caption="The linear gradient with the most color stops.",
+  caption="The linear gradient with the most color stops foudn in our analysis.",
```

Comment:
Removing as it's redundant to disclaim that the results are what we found in our analysis. That could apply to every stat in the chapter! :)

```suggestion
  caption="The linear gradient with the most color stops.",
```